openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -11,9 +12,61 @@ if (typeof gettext === 'undefined') {
         notificationWidget($);
         initNotificationDropDown($);
         initWebSockets($);
+        notificationWindow.clearOldWindows();
+        notificationWindow.add();
     });
+
```

Comment:
```suggestion

       $(window).on('focus', function () {oWNotificationWindow.set();});
```

This will return the authority of notifications to be played to the last window which had focus instead of the last opened window. With the current implementation I observed that the notification sound was getting played in the different window even when I had currently focused on another window.